tendermint/src/amino_types/ed25519.rs

```diff
@@ -140,21 +149,21 @@ mod tests {
 
     #[test]
     fn test_into() {
-        let raw_pk: [u8; PUBLIC_KEY_SIZE] = [
-            0x79, 0xce, 0xd, 0xe0, 0x43, 0x33, 0x4a, 0xec, 0xe0, 0x8b, 0x7b, 0xb5, 0x61, 0xbc,
-            0xe7, 0xc1, 0xd4, 0x69, 0xc3, 0x44, 0x26, 0xec, 0xef, 0xc0, 0x72, 0xa, 0x52, 0x4d,
-            0x37, 0x32, 0xef, 0xed,
+        let raw_pk: [u8; PUBLIC_KEY_LENGTH] = [
+            0xaf, 0xf3, 0x94, 0xc5, 0xb7, 0x5c, 0xfb, 0xd, 0xd9, 0x28, 0xe5, 0x8a, 0x92, 0xdd,
+            0x76, 0x55, 0x2b, 0x2e, 0x8d, 0x19, 0x6f, 0xe9, 0x12, 0x14, 0x50, 0x80, 0x6b, 0xd0,
+            0xd9, 0x3f, 0xd0, 0xcb,
```

Comment:
Ed(wards)25519 point decompression was failing for this? (the old implementation didn't attempt to decompress points)

Reply:
I need to double check compatibility with the go-code here. 

I assume it's OK to check with the upcoming 0.34 / master instead of 0.33 (cc @brapse @ebuchman).

Reply:
OK, I've tested against a few freshly generated pub-keys and they all pass! I think that `raw_pk` wasn't generated using `ed25519.GenPrivKey().PubKey()` and we didn't notice before as the old lib didn't attempt to decompress points.